hera_mc/tests/test_qm.py

```diff
@@ -183,11 +180,11 @@ def test_update_qm_list(self):
         r = self.test_session.get_metric_desc()
         results = []
         for result in r:
-            self.assertTrue(result.metric in self.metrics_dict.keys())
+            self.assertTrue(result.metric in list(self.metrics_dict.keys()))
```

Comment:
This could just be `self.assertTrue(result.metric in self.metrics_dict)`.

Reply:
thanks!